recipes/libtar/conda_build_config.yaml

```diff
@@ -0,0 +1,2 @@
+CONDA_BUILD_SYSROOT:
+  - /opt/MacOSX10.9.sdk        # [osx]
```

Comment:
You should remove this file, in conda-forge it should be configured automatically for you.

Reply:
Ok, but how do I then test local builds?

Reply:
Ah, didn't know I could put it in my home dir, I will remove it in the next commit.